src/.vuepress/theme/layouts/GlobalLayout.vue

```diff
@@ -50,6 +50,7 @@ export default {
   },
 
   setup (_, { root, refs }) {
+    console.log(root.$themeLocaleConfig.skipTo)
```

Comment:
I think you forgot to remove `console.log`

Reply:
Sorry!

Reply:
@kddeisz I fixed it!